src/http/HTTPService.js

```diff
@@ -1,9 +1,7 @@
 const _compact = require('lodash.compact');
-const _map = require('lodash.map');
 const _invokeMap = require('lodash.invokemap');
```

Comment:
I believe the `_invokeMap` usage in this file could be replaced with
```javascript
return self._pluginInstances.map(plugin => plugin.end());
```

Reply:
I believe so but it's not low-hanging fruit. I will take a look at it in the next set of PRs. Coverage is not great to catch is so I don't want to commit to writing additional tests and delaying the main need of the project right now.